### PR TITLE
Complete features changes based on MVP

### PR DESCRIPTION
**Pull request by ellenpham**
I have amended the features/functionality based on what we have discussed yesterday and from what I've been researching today. 

You might need clarification on users' functionality. Below are the 2 main points to discuss:

- `Users can still have full CRUD operations on their account` --> I think we should keep this as nice-to-have as well so we can focus more on the order process for users and the products/categories/orders management for admins (these are more basic features of a e-commerce website).

- Also, users should have Create and Read operations on orders only (not Update and Delete), because once an order is placed (that is Create operation), they cannot modify and delete the order (in reality, they can do cancel orders but we don't go that far in our project). If you were thinking something like users can create, update, delete products in shopping cart, that's different and we don't do Cart API, we saved cart info in our session. 

Please review.

### CODE REVIEW

**Comment by ellenpham** [comment]
Hi all, 

I just made the changes based on what we've discussed this morning and changed the format of Readme file based on the assignment requirements. 

Please review.

File: README.md [hunk 1]
```diff
@@ -1,128 +1,163 @@
...
+### MVP Features
+
+- User authentication and authorisation (Register and Login)
+  - Allow users to create account and login.
+- Password recovery
+  - Allow users to reset password with reset link sent via email.
+- Site navigation and responsive design
+  - Intuitive and responsive UI/UX design for storefront to attract and engage audience.  
+- Search functionality
+  - Allow users to search products by name or by categories.
```
Comment (irenenguyen1017) [inline README.md]: Could you please update the label from 'search products by name' to 'search product by keywords'? With this change, users will be able to search for products that match each keyword they type, making it more versatile and relevant.

**Review by irenenguyen1017 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,128 +1,163 @@
...
+- User account view
+  - View user profile
+  - Change password
+  - View order history
+- Admin dashboard:
+  - Summary view: allow admins to view total users, total products, total orders and total revenues.
+  - User management
+  - Product management
+  - Order management
+
```
Comment (irenenguyen1017) [inline README.md:74]: We also have Category management.

**Review by irenenguyen1017 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ellenpham) [reply README.md]: Sure I will fix that

File: README.md (hunk 2 above)
Comment (ellenpham) [reply README.md:74]: I can add Category management here. But just clarify that in our wirefames, we dont have a separate tab for category management right? So how do admin CRUD categories, I dont think we went through that. 

Maybe we can include CRUD functionality for both products and categories in the Products tab? I will check and note this down in the wireframes too.

**Review by Mana12011207 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (irenenguyen1017) [reply README.md:74]: I'll include the category management tab in the wireframe, my apologies for missing that earlier. We've already mentioned it in the sitemap, so it makes sense to have it here as well. Thank you.

**Review by irenenguyen1017 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (irenenguyen1017) [reply README.md:74]: I'll include the category management tab in the wireframe, my apologies for missing that earlier. We've already mentioned it in the sitemap, so it makes sense to have it here as well. Thank you.

File: README.md [hunk 3]
```diff
@@ -1,128 +1,164 @@
...
-- Design Tools:
-  - Draw.io
-  - Miro
-
-## Tasks planning and tracking
-
-View the description of the way tasks are allocated and tracked in the project [here](./task_tracking.md).
-
-Link to project management tool [Trello](https://trello.com/b/TE5Q9ZYj/t3a2-%F0%9F%8C%B8sakura-pantry).
+## T3A2 Full Stack App
```
Comment (irenenguyen1017) [inline README.md]: Screenshot 2023-11-27 at 9 32 25 pm

The first line in a file should be at the top level. Could you please make this change as well?

File: README.md (hunk 2 above)
Comment (ellenpham) [reply README.md:74]: I made the changes already. 

Can you review again?

**Review by irenenguyen1017 — CHANGES_REQUESTED** [review]

**Review by irenenguyen1017 — APPROVED** [review]

**Pull request merged**